server/middlewares/multer.middleware.js

```diff
@@ -2,32 +2,24 @@ import fs from "fs/promises";
 import path from "path";
 
 import multer from "multer";
+import { v4 as uuidv4 } from "uuid";
 
 import HttpResponseWrapper from "../global/HttpResponseWrapper.js";
 import { addErrorTitle } from "../global/Logger.js";
 
-const MIME_TYPES = {
-  "text/csv": "csv",
-  "application/vnd.ms-excel": "csv",
-};
-
 /**
  * Multer configuration
  */
 const storage = multer.diskStorage({
   filename: (req, file, callback) => {
-    const filename = String(Date.now());
-    const extension = MIME_TYPES[file.mimetype];
-
-    req.body._uploadedFileName = filename;
-    req.body._uploadedFileExtension = extension;
+    const extension = file.originalname.split(".").slice(1).pop();
+    const filename = `${Date.now()}.${uuidv4()}.${extension}`;
```

Comment:
It's great to use uuids :+1: Can we keep the original filename (for example the dci) in the final name to figure out what the file is with it's name?

Reply:
Not necessary because the middleware stores all the information about each file, so in the controllers we can retrieve the original name (among other information).